src/components/PersonalDataForm.tsx

```diff
@@ -16,55 +17,64 @@ export default function PersonalDataForm({ register, variant }: PersonalDataProp
         label="Company name"
         variant="standard"
         fullWidth
+        error={!!errors[variant]?.companyName?.message}
       />
+      {errors[variant]?.companyName?.message && <p>{errors[variant]?.companyName?.message}</p>}
```

Comment:
Text field has a prop `helperText` which allows you to display such errors in an accessible way :)

Reply:
added